sql code.txt

```diff
@@ -0,0 +1,2 @@
+INSERT INTO tblcustomers
+VALUES (2, 'khehra', 'jashan', '7018926456', 'jashansherongmail.com');
```

Comment:
Enter correct code here

Reply:
sir i dont  get the point please explain it agian